fedmsg/core.py

```diff
@@ -463,7 +467,7 @@ def _run_socket(self, sock, name, ep, watched_names=None):
                     else:
                         raise ValidationError(msg)
             else:
-                return name, ep, _topic, msg
+                return (name, ep, _topic, msg)
```

Comment:
equivalent true, it's just more explicit this way i think?
i can remove it (and other things you mention :top: if required :+1:)